Dockerfile

```diff
@@ -1,7 +1,48 @@
-FROM 18fgsa/docker-ruby-ubuntu
-RUN apt-get update
+# FROM 18fgsa/docker-ruby-ubuntu
+FROM python:3.6
 
-# Defaults for ENV vairables
+# Install general dependencies and configure locales
+# TODO: Why is this locale stuff necessary?
+# TODO: I see this error: bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+      apt-utils build-essential git curl libssl-dev \
+      libreadline-dev zlib1g-dev libffi-dev locales \
+    && locale-gen en_US en_US.UTF-8 \
+    && dpkg-reconfigure locales
```

Comment:
@jmhooper: what is all this `locale` stuff for? I copied it from the existing Dockerfile(s) since I assume it serves some purpose, but I'm not sure what that is.